### PR TITLE
Backport Helm 3 documentation from 4.5 to 4.2 (#1020)

### PR DESCRIPTION
**Pull request by jodavis-suse**
To support the backport of helm3 (and the update-alternatives
packaging features to the "helm" Helm 2 package), this commit
copies the relevant text back to the CaaS4 branch.

# Describe your changes
This change backports instructions related to using Helm 3 in CaaSP 4.5 and migration from Helm 2 to 3

Note this change is not an exact cut and paste, as the "helm" package in 4.2.4 was not renamed to "helm2".

# Related Issues / Projects

Relates to:
SUSE/avant-garde#1949

### CODE REVIEW

**Comment by jodavis-suse** [comment]
These documentation changes are required when SUSE/avant-garde#1949 merges.

**Review by r0ckarong — CHANGES_REQUESTED** [review]
Some minor fixes, rest looks good so far.

File: adoc/admin-software-installation.adoc [hunk 1]
```diff
@@ -98,18 +98,38 @@ sudo zypper in ceph-common xfsprogs
 [[helm_tiller_install]]
 === Installing Helm
 
-As of {productname} {productversion},, Helm is part of the {productname} package repository, so to use this,
+As of {productname} {productversion}, Helm is part of the {productname} package repository, so to use this,
```
[the remark this answers is not in the document]
Comment (jodavis-suse) [reply adoc/admin-software-installation.adoc:101]: We introduced it in CaaSP 4.5.0, but here we are introducing it to the 4.2.x branch at 4.2.4 as a backport.  So you are right that we should lock in the version here.

**Comment by r0ckarong** [comment]
Manually picked to `update-424` branch.

**Pull request closed without merging**